src/common/doc/installation.h

```diff
@@ -131,10 +131,10 @@ make \endverbatim
  *
  * \subsection eigen Eigen
  * Some GUDHI modules (cf. \ref main_page "modules list"), and few examples require
- * <a target="_blank" href="http://eigen.tuxfamily.org/">Eigen</a> is a C++ template library for linear algebra:
+ * <a target="_blank" href="https://eigen.tuxfamily.org/index.php?title=Main_Page">Eigen</a> is a C++ template library for linear algebra:
```

Comment:
I am not convinced that hardcoding the `index.php?title=Main_Page` part of the URL is really an improvement, it makes the link more fragile in case they update their website. Nothing too bad though.

Reply:
```suggestion
 * <a target="_blank" href="https://eigen.tuxfamily.org">Eigen</a> is a C++ template library for linear algebra:
```